mungegithub/run.sh

```diff
@@ -16,7 +16,7 @@
 
 while true; do
     /mungegithub \
-      --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \
```

Comment:
We weren't searching enough events, so we removed for some edge cases what
we should not have...
On Aug 31, 2015 5:10 PM, "Tim Hockin" <notifications@github.com> wrote:

> In mungegithub/run.sh
> <https://github.com/kubernetes/contrib/pull/43#discussion_r38374618>:
>
> > @@ -16,7 +16,7 @@
> >
> >  while true; do
> >      /mungegithub \
> > -      --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \
>
> What's the correlation? I'm missing somethign
> … <#14f863dcf448df1d_>
> On Mon, Aug 31, 2015 at 4:47 PM, Brendan Burns <notifications@github.com>
> wrote: In mungegithub/run.sh <
> https://github.com/kubernetes/contrib/pull/43#discussion_r38373179>: > @@
> -16,7 +16,7 @@ > > while true; do > /mungegithub \ > -
> --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \ yes, see:
> #45 <https://github.com/kubernetes/contrib/pull/45> <
> https://github.com/kubernetes/contrib/pull/45> — Reply to this email
> directly or view it on GitHub <
> https://github.com/kubernetes/contrib/pull/43/files#r38373179>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/contrib/pull/43/files#r38374618>.
>